src/rateEngine/__tests__/rates/data/TOU_EV_7.ts

```diff
@@ -0,0 +1,123 @@
+import { getCommonRateElements } from "./commonRateElements";
+
+// https://www.sce.com/business/rates/electric-car-business-rates/business/rates/electric-car-business-rates
+// https://www.sce.com/sites/default/files/inline-files/TOU-EV-7_8_9%20Rate%20Fact%20Sheet_WCAG%20(2).pdf
+// https://edisonintl.sharepoint.com/teams/Public/TM2/Shared%20Documents/Forms/AllItems.aspx?id=%2Fteams%2FPublic%2FTM2%2FShared%20Documents%2FPublic%2FRegulatory%2FTariff%2DSCE%20Tariff%20Books%2FElectric%2FSchedules%2FGeneral%20Service%20%26%20Industrial%20Rates&p=true&ga=1
+//
+// Service under this Schedule will be supplied at one standard voltage.
+const touEV7 = (year: number) => {
+  return {
+    name: "TOU-EV-7",
+    title: "20kW or Less.",
+    code: "TOU_EV_7",
+    helpText:
+      "Applicable to businesses that separately meter the charging of their electric vehicles with charging demands of 20 kilowatts (kW) or less.",
+
+    minKw: 0,
+    maxKw: 20,
+
+    rateElements: getCommonRateElements("TOU_EV_7", years, year),
+  };
+};
+
+// * NOTE: EV-7 rates are the same for each year and have 0 demand charges
+const years = {
```

Comment:
I'm assuming this is just copied in straight from the SCE Fleets codebase. This structure is very specific to that application and IMO doesn't belong in the library (it would just be confusing to a random person trying to digest this).

could we simplify the rate mocks into a single year?